components/header.js

```diff
@@ -11,6 +11,7 @@ export const Header = () => {
             </div>
             <div className={styles.main}>
                 <div onClick={() => router.push('/')}>Home</div>
+                <div onClick={() => router.push('/news/1')}>News</div>
                 <div onClick={() => router.push('/support')}>Support</div>
                 <div onClick={() => router.push('/wins')}>Wins</div>
```

Comment:
Just want to leave this as a pin! But I'm thinking that we should do some more research on using `Link` from NextJS instead of pushing to the route. It might not be a big deal but I'm wondering if one would be better than the other or what the difference would be if we switched back to `Link`

Reply:
Oooh, we should probably switch to `Link`! Here's a stackoverflow that compares link, router.push, and <a> tags. https://stackoverflow.com/questions/65086108/next-js-link-vs-router-push-vs-a-tag